packages/cli/src/seeder-daemon

```diff
@@ -8,6 +8,12 @@ const seederDaemon = async () => {
   const sdk = new SDK(config)
 
   await sdk.start()
+
+  process.on('message', function (packet, handler) {
```

Comment:
We need to removeListeners to not leak. Maybe on some destroy/close step?

Reply:
Done (on `process.exit`)